### ISSUE TITLE
RFE: support arduino build

### ISSUE THREAD

**Issue by akostadinov** [latchset/tang#90]
Hello, tang is most useful as a small, dedicated, energy efficient device. Arduino fits very well. Any chances tang can be compiled for it?

**Comment by er1z** [comment]
This is also one of thoughts I came across once I met this project.

**Comment by sarroutbi** [comment]
Hello. Sorry but this is not planned in the short term. However, we welcome a PR or at least a more detailed description on how to do it:
* is it restricted to cross-compilation?
* What about storage for keys?
* Are dependencies (libjose, jose, libjansson, libhttp_parser) already cross-compiled?

**Comment by er1z** [comment]
Background: home-lab where running another server costs energy.
Storage: EEPROM or SDCard via SPI.
Cross-compilation: can you rephrase?

Potential solution: as ESP-family is quite wide-spread (and there are ESP32 with ethernet) may be it's possible to at least create (Micro)Python binding to expose the functionality?

**Comment by akostadinov** [comment]
Exactly - energy, noise, space. It can be a one function device hidden away from the server(s). This is how I have always imagined it - like a security key, then the rest of the servers just work or not depending on its presence.

To install a whole machine for such small server is a great overkill.

**Comment by n80fr1n60** [comment]
@akostadinov  maybe that helps - i know long time for an answer
https://github.com/latchset/tang/pull/153

**Comment by ghost** [comment]
related https://github.com/latchset/tang/issues/148
